metrics/README.md

```diff
@@ -3,6 +3,20 @@
 This section contains a collection of scripts designed to measure the performance
 and responsiveness of the different components which form Clear Containers.
 
+> **WARNING**
+>
+> Running the metrics tests may modify your existing setup or containers.
+>
+> During initialisation, the metrics tests code tries to bring the system into a
+> known stable state before executing the tests. This action involves removing
+> and killing off any running containers, any parts of the Clear Containers
+> runtime system that may have been left running from any previous executions,
+> and resetting some sub-parts of the container subystems (such as docker) via
+> `systemctl` et. al.
+>
+> It is therefore recommended you do not run the metrics tests on a 'live' system
```

Comment:
Lines 17-18 suggested rewrite: We recommend that you do not run the metrics test on a "live" system that has active containers you do not want to kill.